scripts/deploy_local.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
this was causing really difficult-to-diagnose issues for me. it turns out that `sh` is aliased to `bash` on most systems (but not on mine, where it's actually `sh`), and the script was using some `bash` syntax here and there.